### PR TITLE
Bug/tifffile wo axes tags

### PR DESCRIPTION
**Pull request by k-dominik**
this fixes issue https://github.com/ilastik/ilastik/issues/1487

basically copied the axis order inference from `OpStreamingHdf5Reader`. Also added a test.

### CODE REVIEW

**Comment by k-dominik** [comment]
Ok, I think I might have forgotten to push my latest commit. I'm afraid it'll have to wait until Monday...

**Comment by k-dominik** [comment]
Okay. I have looked into this again. The test fails on circleCI because dependencies for `ilastik-deps-headless` (on circleCI) and `ilastik-everything-no-solvers` (on my local machines) are not in sync. Particularly the versions of scikit-image differ (and also tifffile) (0.13.0 (remote) and 0.12.3 (local)). In the newer version, on circle-CI, the file is created with valid axes tags ('tyx') and the problem doesn't occur in the first place.
 
In conclusion, before merging this PR the following things should be done

For `testOpTiffReader.py`:
- [ ] Write the test such that it checks for the scikit-image version and always saves a file with unknown axis tags
- [ ] Test all different `ndims`

and also 

- [ ] the code to determine the axis-order from `ndims` can be found multiple times in lazyflow. This should be moved to a common place (`lazyflow.utility`?!)

File: lazyflow/utility/helpers.py [hunk 1]
```diff
@@ -250,7 +250,45 @@ def __iter__(self):
...
+
+    Returns:
+        str: String, each position represents one axis.
+    """
+    axisorders = {
+        2: 'yx',
+        3: 'zyx',
+        4: 'zyxc',
+        5: 'tzyxc'
+    }
```
Comment (stuarteberg) [inline lazyflow/utility/helpers.py]: I'm not sure, but I think it's more common to see tiff files that use `cyx` order instead of interleaved channels (`yxc`).  Similarly, I think `tzcyx` or `tczyx` might be more common than ilastik's internal default of `tzyxc`.

(I wonder if we can find any example multi-channel files in the wild to corroborate or dispute this claim of mine....)

Comment (k-dominik) [inline lazyflow/utility/helpers.py]: Hi @stuarteberg,

thank you for looking at the PR.

I have used what I have found in the lazyflow code, so the default behaviour _now_ is unchanged.
At least we can control it from a single point now. Should we leave the PR open until we can have a discussion on that?

Or, would you say, that default axis ordering depends on the file type and should be handled more flexibly?

Comment (akreshuk) [inline lazyflow/utility/helpers.py]: > At least we can control it from a single point now.
Amen to that. 

We want to use tzyxc internally, but how exactly we interpret the input is open. I just read some random big tiff i found on my hdd with vigra, it turned out as xyzc. Another one with colors was xyc, but maybe it's vigra default order? How would you check the order "in the wild"? Interestingly, if you try to read a 2D RGB image with vigra.readVolume, it just eats RAM and hangs. I wonder what causes that.
